packages/typescript/cli/ts-build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 set -o xtrace
-exec tsc -p ./tsconfig.prod.json
+tsc -p ./tsconfig.prod.json && test -f ./tsconfig.browser.json && tsc -p ./tsconfig.browser.json
```

Comment:
I think we should keep the `exec`s there:
https://github.com/ethereumjs/ethereumjs-config/pull/13#pullrequestreview-192175668

Reply:
Is the idea to silently fail when there's no browser config? If so, 👍

Reply:
re: exec I was having issues with it, I will dig deeper for its purpose, thanks for the link

Reply:
Removing the exec is right, as it means "replace the bash process with this other one". So the script will stop after `tsc` executes, even if it's successful. For example, this never prints Hi:

```bash
exec true
echo "Hi"
```

What I'd do is `set -e` at the beginning of the script. That makes the bash process fail if any of the commands fail.